src/foam/u2/TextInputCSS.js

```diff
@@ -16,7 +16,7 @@ foam.CLASS({
       name: 'code',
       value: `
         ^ {
-          height: %INPUTHEIGHT%;
+          height: /*%INPUTHEIGHT%*/ 32px;
```

Comment:
fixes an issue with the country drop down not rendering to the same size as other fields.